packages/pigeon/platform_tests/ios_unit_tests/ios/RunnerTests/AllDatatypesTest.m

```diff
@@ -55,24 +56,25 @@ - (void)testAllEquals {
   everything.aMap = @{@"hello" : @(1234)};
   everything.mapWithObject = @{@"hello" : @(1234), @"goodbye" : @"world"};
   EchoBinaryMessenger *binaryMessenger =
-      [[EchoBinaryMessenger alloc] initWithCodec:FlutterEverythingGetCodec()];
-  FlutterEverything *api = [[FlutterEverything alloc] initWithBinaryMessenger:binaryMessenger];
+      [[EchoBinaryMessenger alloc] initWithCodec:FlutterIntegrationCoreApiGetCodec()];
+  FlutterIntegrationCoreApi *api =
+      [[FlutterIntegrationCoreApi alloc] initWithBinaryMessenger:binaryMessenger];
   XCTestExpectation *expectation = [self expectationWithDescription:@"callback"];
-  [api echoEverything:everything
-           completion:^(Everything *_Nonnull result, NSError *_Nullable error) {
-             XCTAssertEqual(result.aBool, everything.aBool);
-             XCTAssertEqual(result.anInt, everything.anInt);
-             XCTAssertEqual(result.aDouble, everything.aDouble);
-             XCTAssertEqualObjects(result.aString, everything.aString);
-             XCTAssertEqualObjects(result.aByteArray.data, everything.aByteArray.data);
-             XCTAssertEqualObjects(result.a4ByteArray.data, everything.a4ByteArray.data);
-             XCTAssertEqualObjects(result.a8ByteArray.data, everything.a8ByteArray.data);
-             XCTAssertEqualObjects(result.aFloatArray.data, everything.aFloatArray.data);
-             XCTAssertEqualObjects(result.aList, everything.aList);
-             XCTAssertEqualObjects(result.aMap, everything.aMap);
-             XCTAssertEqualObjects(result.mapWithObject, everything.mapWithObject);
-             [expectation fulfill];
-           }];
+  [api echoAllTypes:everything
```

Comment:
again

Reply:
There are more in other tests as well, but I'm going to assume it was intentional and stop pointing them all out.

Reply:
I had decided not to bother changing all the variable names; `everything` is still a fine name since everything (or close to it) is populated in the class.

Reply:
(And the type isn't generally a good name for a variable anyway, so `allTypes` would still not be a great name.)